arches/app/src/arches/declarations.d.ts

```diff
@@ -8,6 +8,7 @@ declare module "cross-env";
 declare module "cytoscape-cola";
 declare module "font-awesome";
 declare module "jquery-validation";
+declare module "js-cookie";
```

Comment:
I think it would be better to add the types package from DefinitelyTyped to the devDependencies repo rather than ignore them here.

Reply:
Oh, we've actually already done that, so I think we won't need this change it if we take the other approach...